symex-interface-builtins.el

```diff
@@ -0,0 +1,37 @@
+;;; symex-interface-builtins.el --- Builtin symex interface extensions -*- lexical-binding: t; -*-
+
+;; URL: https://github.com/countvajhula/symex.el
```

Comment:
I don't have a strong opinion on this module, and keeping it seems marginally useful. But this URL should be updated to `...drym-org/symex.el`